cv_next/lib/definitions.ts

```diff
@@ -8,13 +8,44 @@ export default class Definitions {
 }
 
 export const heroHeader: PageHeader = {
-  header: `Landing pages made easy`,
-  subheader: `Easy to setup. Customizable. Quick. Responsive.`,
-  image: `/hero-img.webp`,
+  header: `Next`,
+  image: `/images/logo.png`,
+  subheader: `improve the magshimim next community's resumes.`,
+  explenetion: `we as a magshimim next community can help each other get jobs by improving our memeber's resumes.
+  this website is a place to uplode resumes, let other review them and make them better.
+  we will not tolerate any abuse, hateful commects, griffing, pentesting and etc... `
 }
 
 export const inactiveHeader: PageHeader = {
   header: `Access Denied`,
   subheader: `This page requieres that you get approved by the moderators`,
   image: `/access-denied.webp`,
 }
+
+
+export const routes: routes[] = [
+  {
+    route: "Home",
+    path: "/",
+    image: '/images/home.png',
+    UILocation: 'navbar'
+  },
+  {
+    route: "Feed",
+    path: "/feed",
+    image: '/images/feed.png',
+    UILocation: 'navbar'
+  },
+  {
+    route: "Login",
+    path: "/login",
+    image: '',
+    UILocation: 'profile'
+  },
+  {
+    route: "Signout",
+    path: "/logout",
```

Comment:
redirect to home on signout, no actual page for that

Reply:
you wanna me to do /Home?

Reply:
yep that works(how the sign out operates at least in the context of the popup is that clicking on it is caught and `supabase.auth.signout()` is called

Reply:
ok so ill do /Gome, you see what i did? its like go home